app/scss/adminPage.scss

```diff
@@ -122,4 +123,24 @@ body {
 }
 .top-grade {
   background-color: $light-purple;
+}
+
+.new-test-container {
+  margin-top: 30px;
+}
+
+#addTestName {
+  margin-bottom: 10px;
+}
+
+.new-test-button {
+  margin-top: -4px;
```

Comment:
this is not an okay way of solving this issue. the button should be the same height as the input. the other buttons are, why isnt this one?